frontend/src/Map.js

```diff
@@ -98,11 +122,12 @@ class Map extends Component {
             // Important! Always set the container height explicitly
             <Grid item xs={8}>
                 <Paper variant="outlined">
-                    <div style={{ height: '96vh', width: '100%' }}>
+                    <div className="map-wrapper">
                         {mapsApiLoaded && <FarmSearchBox map={mapInstance} mapsapi={mapsapi} onPlacesChanged={(data, ref) => this.handleSearch(data, ref)} />}
+                        <button className="btn-marker" onClick={() => this.getRedMarkerBack()}><FontAwesomeIcon icon={faMale} /></button>
                         <GoogleMapReact
                             bootstrapURLKeys={{
-                                key: '',
+                                key: 'AIzaSyB4iv4WFatScZofidFmwL7btq3TZVcXGbo',
```

Comment:
Is this a password? If yes, maybe this is not the best way to store it.

Reply:
fixed